perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -860,13 +860,19 @@ sub report_edit_location : Private {
 
     if ( $c->stash->{latitude} != $problem->latitude || $c->stash->{longitude} != $problem->longitude ) {
         $c->log->debug("lat/lon have changed");
+        $c->log->debug("stash->{bodies_to_list}: " . Dumper($c->stash->{bodies_to_list}));
+        delete $c->stash->{prefetched_all_areas};
+        delete $c->stash->{all_areas};
+        delete $c->stash->{fetch_all_areas};
+        delete $c->stash->{all_areas_mapit};
```

Comment:
Hmm, do we need a nicer way of causing a 'fresh' lookup in some way?
Should it check whether load_and_check_areas errored at all (e.g. drag pin outside area entirely, I guess?)

Some debug stuff left in around here.